<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPlainTextEdit" name="write_field">
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="current_file">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>563</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="file_menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="open_btn"/>
    <addaction name="save_btn"/>
    <addaction name="append_btn"/>
    <addaction name="close_btn"/>
   </widget>
   <widget class="QMenu" name="interface_menu">
    <property name="title">
     <string>Interface</string>
    </property>
    <addaction name="dark_btn"/>
    <addaction name="white_btn"/>
    <addaction name="incr_font_btn"/>
    <addaction name="decr_font_btn"/>
   </widget>
   <addaction name="file_menu"/>
   <addaction name="interface_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="open_btn">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="save_btn">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="append_btn">
   <property name="text">
    <string>Append</string>
   </property>
  </action>
  <action name="dark_btn">
   <property name="text">
    <string>Dark mode</string>
   </property>
  </action>
  <action name="white_btn">
   <property name="text">
    <string>Light mode</string>
   </property>
  </action>
  <action name="close_btn">
   <property name="text">
    <string>Close File</string>
   </property>
  </action>
  <action name="incr_font_btn">
   <property name="text">
    <string>Increment font size(equal Ctrl +)</string>
   </property>
  </action>
  <action name="decr_font_btn">
   <property name="text">
    <string>Decrement font size(equal Ctrl -)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
